A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/engines.cfg

```diff
@@ -19,7 +19,7 @@ fuel_flow_gain = 0.002 ; Gain on fuel flow
 inlet_area = 29 ; Square Feet, engine nacelle inlet area
 rated_N2_rpm = 29920 ; RPM, second stage compressor rated value
 static_thrust = 26550 ; Lbs, max rated static thrust at Sea Level
-reverser_available = 1.5
+reverser_available = 1  ; Modified

```

Comment:
Was this intentional?

Reply:
Forgot to remove that, was going to push it seperately